book/compiler-frontend/README.md

```diff
@@ -1090,7 +1090,7 @@ This module defined bindings for the Ncurses library. First, compile the
 interfaces with `-bin-annot` so that we can obtain the `cmt` and `cmti`
 files, and then run `ocp-index` in completion mode:
 
-```sh dir=../../examples/code,source-tree=../../examples/code/ffi,skip
+```sh dir=../foreign-function-interface/examples,source-tree=../foreign-function-interface/examples/ffi,skip
```

Comment:
`compiler-frontend` uses an example from `foreign-function-interface`.

Reply:
My take on this is that we should duplicate them now thus allowing them to diverge in the future. I think depending on files that aren't within the current directory doesn't cope well with `dune` and it makes maintining the chapters more complex than it should.

Any thought on that @yminsky @avsm ?

Reply:
Duplicating is just fine in this case.

Reply:
I wonder if duplicating will cause confusion. The text before this block says "Let's refer back to our Ncurses binding example from  [...]" and the block starts with `(cd ffi/ncurses [...]` (this path may have to be updated). Readers should be able to find the source code easily.

Reply:
I think it'd be okay to duplicate. If we want the code to be strictly identical we can enforce it with a dune `diff` rule for the duplicates in `book/dune` for instance.

No solution is perfect, even the current one as if we rewrite a chapter A without removing the example it's still possible that a later chapter B references out of date examples from A and it wouldn't be enforced by the CI anyway.